spec/cypress/integration/notifications/SignInEmail.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Sign In Email
```

Comment:
In my opinion, sign in email isn't a feature - sign in is, and email is part of that

Reply:
Cool - I can change the name

Reply:
Done